lib/react_dom.dart

```diff
@@ -0,0 +1,27 @@
+library react_dom;
+
+/// Renders a ReactElement into the DOM in the supplied container and return a reference to the component
+/// (or returns null for stateless components).
+///
+/// If the ReactElement was previously rendered into the container, this will perform an update on it and only
+/// mutate the DOM as necessary to reflect the latest React component.
+var render = (component, element) {
```

Comment:
Should the `element` param be renamed `container`? Would be nice if the dartdoc comment could reference these params by `[name]`